03-specifying-sample-designs.Rmd

```diff
@@ -534,9 +534,9 @@ des_fay <- dat %>%
 
 #### Example {-} 
 
-The 2015 RECS uses Fay's BRR weights with the final weight as NWEIGHT and replicate weights as BRRWT1 - BRRWT96 with $\rho=0.5$^[Using the 2015 microdata file to compute estimates and standard errors (RSEs): https://www.eia.gov/consumption/residential/data/2015/pdf/microdata_v3.pdf]. On the file, DOEID is a unique identifier for each respondent, TOTALDOL is the total cost of energy, TOTSQFT_EN is the total square footage of the residence, and REGOINC is the Census region.
+The 2015 RECS uses Fay's BRR weights with the final weight as NWEIGHT and replicate weights as BRRWT1 - BRRWT96 with $\rho=0.5$^[Using the 2015 microdata file to compute estimates and standard errors (RSEs): https://www.eia.gov/consumption/residential/data/2015/pdf/microdata_v3.pdf]. On the file, DOEID is a unique identifier for each respondent, TOTALDOL is the total cost of energy, TOTSQFT_EN is the total square footage of the residence, and REGOINC is the Census region. 
 
-To load any data in the book, use the function `read_rds_tsr()`, which reads the data file from a directory on OSF where all book data has been placed for sharing. 
+To load any data in the book, use the function `read_rds_tsr()`, which reads the data file from a directory on OSF where all book data has been placed for sharing.  
```

Comment:
Let's move this to the set up section of this chapter, or some overview in chapter 1. Thoughts?